.travis.yml

```diff
@@ -1,16 +0,0 @@
-sudo: false
-language: java
-env:
-  global:
-    - GRADLE_OPTS="-Xmx512m"
-jdk:
-  - oraclejdk8
-script:
-  - ./gradlew check --info
-  - ./testutils/check-baselines.sh
-  - go test testutils/check-calling-form-in-baseline/*.go
-  - go run $(find testutils/check-calling-form-in-baseline -type f | grep -v _test) -yaml src/test/java/com/google/api/codegen/testsrc/libraryproto/library_gapic.yaml src/test
-  - go test testutils/check-lang-uses-form/*.go
-  - go run $(find testutils/check-lang-uses-form -type f | grep -v _test) -forms src/main/java/com/google/api/codegen/viewmodel/CallingForm.java src/test
-after_success:
-  - bash <(curl -s https://codecov.io/bash)
```

Comment:
seems we'll lose the coverage? Should we also do this in Circle?

Reply:
Derp. Fixed now. The report is below.